src/themes/ec/variables/_content-item.scss

```diff
@@ -2,8 +2,9 @@
 @use '../index' as *;
 
 $content-item: (
-  divider-color: var(--c-n),
+  divider-color: var(--cm-border-neutral-lowest),
   padding: var(--s-xl),
-  image-border-color: var(--c-n),
+  image-border-color: var(--cm-border-neutral-lowest),
```

Comment:
mmh..based on the reply by the designers this border color should follow the color mode, border-neutral doesn't, guess that it should be --cm-border-low

Reply:
thanks, also removed the background for the image.